### PR TITLE
Add release notes and updates for 1.11.1

### PR DESCRIPTION
**Pull request by morphis**
Component versions still need to be updated as soon as they are pushed live.

### CODE REVIEW

File: announcements/1.11.1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.1
```
Comment (ru-fu) [inline announcements/1.11.1.md:3]:
```suggestion
The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.1.
```

File: announcements/1.11.1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New Features & Improvements
+
+ * Android security updates for August 2021 (see [here](https://source.android.com/security/bulletin/2021-08-01) for more information)
```
Comment (ru-fu) [inline announcements/1.11.1.md:9]:
```suggestion
 * Android security updates for August 2021 (see [Android Security Bulletin - August 2021](https://source.android.com/security/bulletin/2021-08-01) for more information)
```

"here" is generally a really bad link text. ;)

File: announcements/1.11.1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* LP #1939277 lxc-attach fails on sendfile with EINVAL on 5.11
+* LP #1938877 Native crash occurred when creating an application from Android 11 after finishing application bootstrap
+* LP #1939274 Anbox crashes after "Failed to put memory protection in place"
+* LP #1939666 Bootstrap fails because of missing /dev/fd0
+* LP #1939129 The anbox-stream-sdk.js file is missing from Android webview based projects
+* LP #1938901 Appliance upgrade fails with Juju 2.9.x
+
+#### Upgrade Instructions
+
+Please see the [general upgrade guide](https://anbox-cloud.io/docs/installation/upgrading-from-previous-versions) for instructions of how to update your Anbox Cloud deployment to the 1.11.1 release.
```
Comment (ru-fu) [inline announcements/1.11.1.md:22]:
```suggestion
See the [general upgrade guide](https://anbox-cloud.io/docs/installation/upgrading-from-previous-versions) for instructions on how to update your Anbox Cloud deployment to the 1.11.1 release.
```

File: component-versions.md [hunk 4]
```diff
@@ -2,6 +2,60 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+
+### Snaps
+
+| Name | Version |
+|----------|--------------|
+| ams    | 1.11.1 |
+| aar    | 1.11.1 |
+| ams-node-controller    | 1.11.1  |
+
+## Debian Packages
```
Comment (ru-fu) [inline component-versions.md:44]: Is this supposed to be an h2 heading (with h3's above)?

File: release-notes.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
 > **NOTE**: If you're interested in getting notified for the latest Anbox Cloud releases, make sure you subscribe to notifications on the [announcements  category](https://discourse.ubuntu.com/c/anbox-cloud/announcements/55) on the Anbox Cloud discourse.
 
+[Details=1.11.1]
+
+# 1.11.1 (August 17 2021)
+
+#### New Features & Improvements
+
+ * Android security updates for August 2021 (see [here](https://source.android.com/security/bulletin/2021-08-01) for more information)
```
Comment (ru-fu) [inline release-notes.md:9]:
```suggestion
 * Android security updates for August 2021 (see [Android Security Bulletin - August 2021](https://source.android.com/security/bulletin/2021-08-01) for more information)
```

**Review by ru-fu — COMMENTED** [review]
Just some small nitpicks.

File: component-versions.md (hunk 4 above)
Comment (morphis) [reply component-versions.md:44]: Ah no, let me correct that.

Comment (morphis) [reply component-versions.md]: Done

**Review by ru-fu — APPROVED** [review]

**Pull request merged**